modules/ocm-nw-networkpolicy-view.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * networking/network_policy/viewing-network-policy.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-networkpolicy-view-ocm_{context}"]
+= Viewing network policies using {cluster-manager}
+
+You can view the configuration details of your network policy in {cluster-manager-first}.
+
+.Prerequisites
+
+* You logged in to {cluster-manager-url}.
+* You created an {product-title} cluster.
+* You configured an identity provider for your cluster.
+* You added your user account to the configured identity provider.
+* You created a network policy.
+
+.Procedure
+
+. From the *Administrator* perspective in the {cluster-manager} web console, under *Networking*, click *NetworkPolicies*.
+. Select the desired network policy to view.
+. In the *Network Policy* details page, you can view all of the associated ingress and egress rules.
+. Selecting *YAML* on the network policy details shows the policy configuration in YAML format.
```

Comment:
You could update this step to use parallelism with the other steps and to direct the action to the reader. For example:

```
. Select *YAML* on the network policy details to view the policy configuration in YAML format.
```